### DISCUSSION TITLE
Option to disable alignment

### DISCUSSION THREAD

**Discussion by supaeasy** [jomjol/AI-on-the-edge-device#1806] [General] [answered]
### The Feature

Hi there would it be possible to add an option to avoid alignment altogether? I am having massive issues with this. The picture taken fits perfectly into ROI digit spaces - then alignment kicks in and everything gets messed up. Since I fixated my camera completely I dont really need alignment anyway.

**Comment by caco3** [comment]
I don't think disabling the alignment is the right way to go.
I expect your alignment marks are not chosen ideally!

Can you provide a screenshot showing the alignment marks?

**Comment by supaeasy** [comment, marked as answer]
I solved this. Found that in expert options you can disable it. It still says "aligning" in status but it doesnt distort the Image anymore.

**Comment by nliaudat** [comment]
Option alignment algo off : https://github.com/jomjol/AI-on-the-edge-device/pull/1924
Works great without alignment algo and round is faster.
